src/DogStatsd.php

```diff
@@ -155,6 +157,28 @@ public function __construct(array $config = array())
         );
 
         $this->resetTelemetry();
+
+        $originDetectionEnabled = $this->isOriginDetectionEnabled($config);
+        if ($originDetectionEnabled) {
+            $originDetection = new OriginDetection();
+            $containerID = isset($config["container_id"]) ? $config["container_id"] : "";
```

Comment:
I may be missing something, but this doesn't seem to match behavior of the Go client, where user provided containerId is always honored.